README.md

```diff
@@ -4,12 +4,13 @@ Meatpocalypse
 Before attemping to run the game at all, make sure you have Node v5.8.0 installed.
 
 The following tools and API's are needed:
-- sqlite3 (database)
-- express (web server)
+- sqlite3
+- express
+- body-parser
 
 To run the game, they must all be installed:
 
-``` $ sudo npm install sqlite3 express ```
+``` $ sudo npm install sqlite3 express body-parser ```
```

Comment:
btw if you do `npm install body-parser --save`, it will add it to the `package.json` file. This file lists all the dependencies formally, and can be install via `npm install`. That's idiomatic in nodejs. I'm assuming inspired by other package managers, like ruby's bundler.